docs/source/dev_guide/release.rst

```diff
@@ -81,7 +81,7 @@ Releasing Anaconda
 ------------------
 
 1. Be sure to have already completed :ref:`Releasing On GitHub <github-release>`. This triggers the CI/CD workflow that handles Anaconda releases.
-2. Wait until the CI/CD build is successful. You can view all workflows at `All Workflows <https://github.com/E3SM-Project/zppy/actions>`_.
+2. Wait for a bot PR to come up automatically on conda-forge after the GitHub release.
 3. Check the https://anaconda.org/e3sm/zppy page to view the newly updated package.
```

Comment:
Is there a new page to point to?

Reply:
> The conda-forge bot PR can come as quickly as an hour later or as late as a day later 

> The conda-forge ones will be available at anaconda.org/conda-forge/zppy

The most recent commit addresses these.

> if you want a quicker release

I'm not sure how quickly @xylar will need releases. Usually when we're testing E3SM Unified, we want to get release candidates integrated pretty quickly so we can start testing them. I suppose a day is a reasonable amount of time?

> Do you still intend to release on the e3sm channel?

I don't think so? I think one channel is probably going to be sufficient.